code/molecular_phenotypes/calling/splicing_calling.ipynb

```diff
@@ -306,7 +309,7 @@
     "task: trunk_workers = 1, trunk_size = job_size, walltime = walltime, mem = mem, cores = numThreads\n",
     "bash: expand= \"${ }\", stderr = f'{_output[0]:n}.stderr', stdout = f'{_output[0]:n}.stdout', container=container\n",
     "    cat ${_input} > ${_output:nn}.junc\n",
-    "    leafcutter_cluster_regtools.py -j ${_output:nn}.junc -m ${min_split_reads} -o ${_output:nn} -l ${max_intron_len}"
+    "    python leafcutter_cluster_regtools.py -j ${_output:nn}.junc -m ${min_split_reads} -o ${_output:nn} -l ${max_intron_len}"
```

Comment:
in the docker image, we should copy this script to `/usr/local/bin` and `chmod +x` then use it without having to type `python`

Reply:
As it turns out, I actually don't need to chmod + x and add the python explicitly. will remove the alias statement and the python in the leafcutter.ipynb